RELEASE_GUIDELINES.md

```diff
@@ -0,0 +1,56 @@
+# Module Release Guidelines
+
+## Create a Proposal Issue
+
+To prepare for a new release, create a [new issue](https://github.com/cloudevents/sdk-javascript/issues/new?assignees=&labels=&template=feature-request.md&title=) where the title of the issue cleary reflects the version to be released.
+
+For example: "Proposal for 3.2.0 release", or something similar.  If you are not sure which version is the next version to be released, you can run `npm run release -- --dry-run` to find out what the next version will be.
+
+The body of the issue should be the commits that will be part of the release.  This can be easily accomplished by running a git log command with a defined **range**.  This range should start at the most recent version tag and end at the latest commit in the master branch.
+
+For example:
+
+```
+git log v3.0.1..upstream/master --oneline
+```
+
+This will output all the commits from the 3.0.1 tag to the latest commits in the remote upstream/master branch.
+
+This output should be pasted into the issue as normal text.  This will allow Github to magically turn all commit hashes and PR/Issues numbers to links.
+
+### Get Consensus
+
+Before a release can be finalized, other maintainers should give a +1 or a thumbs up or some other identifying mark that they are good with the changes.
+
+// TODO: should we wait a specified time after the +1's before we create
```

Comment:
I don't think this is necessary. All PRs should only land if a maintainer approved it.